site/guides/tenant-onboarding.md

```diff
@@ -0,0 +1,248 @@
+# Tenant onboarding
+
+We have seen that in large organizations using kubernetes, there is a platform
+team (or infrastructure team) that is responsible for managing the kubernetes
+clusters. Typically a kubernetes cluster is shared by multiple teams to run
+different types of workloads. One of the common use-cases platform teams have
+is onboarding a new tenant on the kubernetes cluster. In this guide, you will
+learn - how you can use kpt to address the tenant use-case. Though this guide
+focuses on the tenant use-case, the pattern for package workflow discussed here
+can be applied to other use cases as well.
+
+**Note:** This guide is inspired by the [kube-common-setup](https://github.com/nghnam/kube-common-setup)
+helm chart.
+
+## Terminology
+
+Before we jump into the tutorial, let’s go over some terminologies that are
+used throughout this guide.
+
+### Tenant
+
+A tenant represents a collection of related infra/app resources and needs to be
+isolated from other resources, for example, a microservice, a workload, a team
+(group of developers) sharing common infra resources.
+
+### Platform team
+
+Central infra team responsible for managing the Kubernetes cluster. Members of
+the platform teams typically have administrative privileges on the Kubernetes cluster.
+
+### App developer team
+
+The team responsible for operating the tenant. Once the tenant is provisioned,
+this team will typically deploy resources (workload, services etc) in the tenant.
+
+## Package Organization
```

Comment:
capitalization question: any reason why organization is capitalized while *developer team* and *team* above are not?

Reply:
That's an oversight. No specific reason :) lower-cased `Organization` to make it consistent.